foundry.toml

```diff
@@ -6,7 +6,7 @@ via_ir = true
 solc_version = '0.8.26'
 evm_version = "cancun"
 
-optimizer_runs = 1_000_000
+optimizer_runs = 1
```

Comment:
can this foundry.toml contain separate contigurations for position descriptor and everything else? I dont mind so much with hardhat because we just use it for testing and never for deploys, but we use this foundry.toml for deploys sometimes so want to make sure its always right